examples/car_control/run.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+
+import numpy as np
+import os
+import spear
+import time
+
+common_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "common"))
```

Comment:
I don't think `common_dir` and `import sys` is needed for anything.

Reply:
removed